workspace/apps/pidp/src/app/features/access/pages/bc-provider-edit/bc-provider-edit.page.ts

```diff
@@ -108,6 +129,39 @@ export class BcProviderEditPage
     this.navigationService.navigateToRoot();
   }
 
+  public onResetMfa(): void {
+    const data: DialogOptions = {
+      title: 'Multi-factor authentication (MFA) reset',
+      bottomBorder: false,
+      titlePosition: 'center',
+      bodyTextPosition: 'center',
+      component: HtmlComponent,
+      data: {
+        content:
+          'At your request we are about to reset your MFA, this action cannot be undone. Are you sure you want to do this?',
+      },
+      imageSrc: '/assets/images/online-marketing-hIgeoQjS_iE-unsplash.jpg',
+      imageType: 'banner',
+      width: '31rem',
+      height: '24rem',
+      actionText: 'Continue',
+      actionTypePosition: 'center',
+    };
+    this.dialog
```

Comment:
Writing a dailog helper like
dialogHelper(component, data, afterCloseCallback) {
}
will be reusable in other components.